src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.growth-table');
+  const appendRowButton = document.querySelector('.append-row');
+  const removeRowButton = document.querySelector('.remove-row');
+  const appendColumnButton = document.querySelector('.append-column');
+  const removeColumnButton = document.querySelector('.remove-column');
+
+  const maxCount = 10;
+  const minCount = 2;
+
+  const updateButtons = () => {
+    const rowCount = table.rows.length;
+    const colCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row and one column initially. Accessing `table.rows[0]` could cause an error if the table is empty. Consider adding a check to ensure the table is not empty before accessing its rows or cells.